packages/web-components/src/card/card.ts

```diff
@@ -0,0 +1,207 @@
+import { attr, css, ElementStyles, observable } from '@microsoft/fast-element';
+import { FASTCard, StartEndOptions } from '@microsoft/fast-foundation';
+import { keyEnter, keySpace } from '@microsoft/fast-web-utilities';
+import { CardAppearance, CardControlSize, CardOrientation } from './card.options.js';
+
+/**
+ * Card configuration options
+ * @public
+ */
+export type CardOptions = StartEndOptions<Card>;
+
+/**
+ * @class Card component
+ *
+ * @remarks
+ * This class extends the FASTCard. a flexible content container
+ */
+export class Card extends FASTCard {
+  public connectedCallback(): void {
+    super.connectedCallback();
+    this.setInteractive();
+    this.updateComputedStylesheet();
+  }
+
+  public disconnectedCallback(): void {
+    super.disconnectedCallback();
+  }
+
+  /**
+   * Stores the computed stylesheet for the card
+   */
+  private computedStylesheet?: ElementStyles;
+
+  /**
+   * @property orientation;
+   * @default filled
+   * @remarks
+   * Determines the orientation of the card
+   */
+  @attr
+  public appearance?: CardAppearance;
+
+  /**
+   * @property orientation;
+   * @default vertical
+   * @remarks
+   * Determines the orientation of the card
+   */
+  @attr
+  public orientation?: CardOrientation;
+
+  /**
+   * @property controlSize
+   * @default medium
+   * @remarks
+   * Determines the size of the card
+   */
+  @attr({ attribute: 'control-size' })
+  public controlSize?: CardControlSize;
```

Comment:
Is there a reason we need to include `control` rather than have this just be `size`?

Reply:
No just a bad habit. The last couple components that I built that were spec'd with a `size` attribute and needed to be changed to control-size.

Fixed.